src/Interactions.tsx

```diff
@@ -87,19 +87,22 @@ export function InteractionManager({ children }: { children: any }) {
       const hits = new Set()
       const intersections = intersect(controller)
 
-      intersections.forEach((intersection) => {
+      const intersection = intersections.find((i) => i?.object)
+      if (intersection) {
```

Comment:
R3F lets users control raycaster filtering via `state.raycaster.filter`. Maybe we should use that raycaster and do the same here.